packages/generator-helper/src/dmmf.ts

```diff
@@ -207,7 +211,7 @@ export namespace DMMF {
     delete = 'delete',
     deleteMany = 'deleteMany',
     groupBy = 'groupBy',
-    count = 'count',
+    count = 'count', // TODO: count does not actually exist, why?
```

Comment:
What do you mean?

This exists https://www.prisma.io/docs/concepts/components/prisma-client/aggregation-grouping-summarizing#count

Reply:
I figured out that count is not an operation in the dmmf but we actually use aggregates for count, so maybe we should remove this as it is technically incorrect (might be used an have an actual meaning - to be checked first).

Reply:
Oh right